src/style.css

```diff
@@ -0,0 +1,93 @@
+/* #region page */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+  font-weight: 300;
+}
+/* #endregion */
+
+/* #region forms */
+.form {
+  display: flex;
+  align-items: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.form-large {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.form-large:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.form-small {
+  height: 42px;
+}
+
+.form-small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+/* #endregion */
+
+/* #region searchbars */
+.searchbar {
+  font-family: Avenir, sans-serif;
+  color: inherit;
+  width: 100%;
+  padding: 0;
+  border: none;
+  outline: none;
+}
+
+.searchbar::placeholder {
+  color: #3d4e61;
+}
+
+.searchbar-large {
+  margin-left: 17px;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.searchbar-large:focus {
+  font-weight: 900;
```

Comment:
If for large and small modifier this value is the same so maybe you can move it to block `:focus` for don`t repeat yourself